.rubocop.yml

```diff
@@ -46,23 +48,42 @@ Style/DotPosition:
 Style/EmptyLineBetweenDefs:
   AllowAdjacentOneLineDefs: true
 
+# Maintain compatibility with older Rubies
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
 # Allow multiline block chains
 Style/MultilineBlockChain:
   Enabled: false
 
+# Indent one level for follow-up lines
+Style/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+# Indent one level for follow-up lines
+Style/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+# TODO: Discuss
+Style/MutableConstant:
```

Comment:
No, I'm for it.;)

It's super-hard for me to answer those comments because I just get an email with a link from github. Which doesn't work, cause it refers to an outdated diff. So I have to click on ALL outdated diffs to find your latest comment. @github in the name of all gods available, please fix that.
Let's continue discussing in top level comments.